.github/workflows/patch-release.yml

```
@@ -0,0 +1,24 @@
+name: patch release
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *'
+env:
+  GO_VERSION: '1.22'
+jobs:
+  patch-release:
+    permissions:
+      contents: none
+    if: startsWith(github.ref, 'refs/heads/release-') && github.ref != 'refs/heads/main'
```

I think this one should be enough:
```suggestion
    if: startsWith(github.ref, 'refs/heads/release-')
```